.vscode/tasks.json

```diff
@@ -2,24 +2,17 @@
 	"version": "2.0.0",
 	"tasks": [
 		{
-			"label": "build",
-			"command": "dotnet",
-			"type": "shell",
-			"args": [
-				"build",
-        "--nologo",
-				"--property:GenerateFullPaths=true",
-				"--consoleloggerparameters:NoSummary"
-			],
-			"group": {
-				"kind": "build",
-				"isDefault": true
-			},
-			"presentation": {
-				"reveal": "silent"
-			},
-			"problemMatcher": "$msCompile"
-		},
+            "label": "build",
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/Test/NodeApi.Test.csproj",
```

Comment:
I'd suggest building the `.sln` instead of a specific project. That way it will build the generator project (which is not referenced by the test project) and any other projects that are added to the solution.

Same for the `watch` task below.